files/jenkins/jobs/PR_BUILDER/config.xml

```diff
@@ -50,7 +50,7 @@
     <triggers>
         <org.jenkinsci.plugins.ghprb.GhprbTrigger plugin="ghprb@1.15-0">
             <spec>H/3 * * * *</spec>
-            <adminlist>CHANGE_ME_to_botname
+            <adminlist>
                 podarok
                 ygerasimov
```

Comment:
Maybe we can cleanup adminList? Anyway all users from our company team are admins.